e2e/files.spec.ts

```diff
@@ -80,15 +89,6 @@ const confirmSharingSettings = async (
   )
 }
 
-const confirmAndClosePermissionsSavedAlert = async (page: Page) => {
-  const permissionsAlert = page.getByRole('alert').filter({
-    has: page.getByText('Permissions were successfully saved to Synapse'),
-  })
-  await expect(permissionsAlert).toBeVisible()
-  await permissionsAlert.getByRole('button').click()
-  await expect(permissionsAlert).not.toBeVisible()
-}
```

Comment:
Pulled this into a new method (`dismissAlert`), so it can be reused across tests